usaspending_api/submissions/management/commands/rm_submission.py

```diff
@@ -26,4 +26,10 @@ def handle(self, *args, **options):
             self.logger.error("Submission with broker id " + str(options["submission_id"][0]) + " does not exist")
             return
 
-        submission.delete()
+        deleted = submission.delete()
+
+        statistics = "Statistics:\n  Total objects Removed: " + str(deleted[0])
+        for model in deleted[1].keys():
+            statistics = statistics + "\n  " + model + ": " + str(deleted[1][model])
```

Comment:
This is fine, though `statistics += "\n  {}: {}".format(model, deleted[1][model]` or  `statistics += "\n  %s: %s" % (model, deleted[1][model])` are more popular Python idioms.